src/main/java/com/github/fishio/settings/Settings.java

```diff
@@ -66,7 +66,9 @@ public SimpleDoubleProperty getSliderProperty(String setting) {
 		SimpleDoubleProperty s = sliderSettings.get(setting);
 		if (s == null) {
 			logger.log(LogLevel.ERROR, "Setting '" + setting + "' not found!");
```

Comment:
Okay, I do agree that at this point in time it's probably not worth the time to fix it. I also find that it makes the code look more crowded. The only thing is that we did fix this pmd issue in other places in our code, so someone else should also comment on whether he agrees with us.
<review